snapcraft/remote/launchpad.py

```diff
@@ -102,7 +102,7 @@ def __init__(
         self._project_name = project_name
 
         self._lp: Launchpad = self._login()
-        self.user = self._lp.me.name
+        self.user = self._lp.me.name  # type: ignore
```

Comment:
```suggestion
        self.user = cast(str, self._lp.me.name)
```

should work too

Reply:
I tried that, but the problem is with `_lp` being incorrectly typed:
```
error: Cannot access member "name" for type "NamedOperation"
   Member "name" is unknown (reportGeneralTypeIssues)
error: "name" is not a known member of "None"  (reportOptionalMemberAccess)
```

I can rewrite it as:
```
me = self._lp.me
self.user = me.name
```
and the same two warnings persist for the first line.

So I left it as-is.

Reply:
Well that's disappointing... Thanks for checking!